### DISCUSSION TITLE
Simultaneously running process

### DISCUSSION THREAD

**Discussion by vtisnado** [directus/directus#2797] [Archive]
Hi guys, first of all, thank you for the great work with this CMS, I really love it and I started using in it couple projects.

Im writing this question because yesterday I started seeing `Failed to load resource: the server responded with a status of 508 ()` error and can't access to the admin panel after login. Luckely, the access to the API is working fine at the moment so I have time to investigate about the error in the admin.

This is the error I got:
<img width="857" alt="Screen Shot 2020-07-24 at 11 42 08 PM" src="https://user-images.githubusercontent.com/1319443/88461765-10b04a00-ce6c-11ea-8908-982fcf134aa4.png">

Checking the Chrome dev toolbar Network, I found the message: **508 Resource Limit Is Reached** so I went to the cPanel of my hosting and everything seems to be fine about bandwith disk space, memory usage, etc. but I saw a message saying that my site was limited due over usage of entry process.

This took my attention because one of the sites have decent traffic, but the other (both with Directus) have no traffic at all, so I wanted to know if the CMS have any internal process or update script that is running continuosly.

<img width="785" alt="Screen Shot 2020-07-25 at 12 54 49 AM" src="https://user-images.githubusercontent.com/1319443/88462549-caf68000-ce71-11ea-8c25-1eb1b7b0aaeb.png">

Thanks in advanced,

**Comment by benhaynes** [comment]
Not sure if this is a system call, or something in your schema, but maybe it's a recursive relational fetch?
